setup.py

```diff
@@ -11,19 +9,6 @@
 here = os.path.abspath(os.path.dirname(__file__))
 
 
-def read(*parts):
-    with codecs.open(os.path.join(here, *parts), "r") as fp:
-        return fp.read()
-
-
-def find_version(*file_paths):
-    version_file = read(*file_paths)
-    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]", version_file, re.M)
-    if version_match:
-        return version_match.group(1)
-    raise RuntimeError("Unable to find version string.")
-
-
```

Comment:
This is unused, and causing issue for mypy.

Reply:
yes indeed old code